ironfish/src/indexers/minedBlocksIndexer.ts

```diff
@@ -272,7 +272,9 @@ export class MinedBlocksIndexer {
     for await (const [hash, block] of iterator) {
       if (block.account === accountName) {
         await this.database.transaction(async (tx) => {
-          await this.sequenceToHashes.del(block.sequence, tx)
+          let hashes = (await this.sequenceToHashes.get(block.sequence)) ?? []
```

Comment:
It looks like you're using null coalescence to default this to [] because it can be undefined, but how can it be undefined?

Reply:
That's a fair point. After looking through the code paths, it shouldn't ever be undefined if a block was added correctly. I think doing `Assert.isNotUndefined()` would be a good replacement.